### ISSUE TITLE
Consistent Terminology Style

### ISSUE THREAD

**Issue by ecorm** [wamp-proto/wamp-proto#197]
While reviewing the RFC, I noticed that terms like "Router" or "Subscriber" don't have a consistent typographical style.

I suggest that italics be only used only once when introducing a new term for the first time. That is the convention used by most technical documents (as well as non-technical publications). Using _italics_ for every 4th _word_ gets _hard_ on the _eyes_. It's even worse when the RFC document is rendered in plain text, where `_underscores_` are used to mark weak emphasis.

I have no problem with role names ("Router", "Subscriber") being capitalized, as long as we're consistent everywhere.

Another stylistic convention applies to WAMP message types, which are in ALL_CAPS. Have I missed any other conventions?

Could we establish the "official" terminology style once and for all? Perhaps we can write it down in the README. Once that's established, I can fix non-conformities as I review the RFC.

**Comment by oberstet** [comment]
I agree, we should have a consistent style .. and a guide. @goeddea maybe you could check how the RFC6455 handles this and come up with a proposal?

**Comment by goeddea** [comment]
Added @ecorm's suggestion of the italics (using underscrores) only on first encounter, capitalization afterwards as a rule. Other than that we only have the rule that message types are all caps, and that we use backticks when talking about message arguments.

Had a look at the WebSocket RFC as well as that for HTTP2, and there is no real consistency between documents. The above rules should be fine.

**Comment by oberstet** [comment]
IETF "rules": we can come up with what works for us. This is the IETF .. what works wins.

My view: I prefer every term used in a text that has special meaning (beyond the general word) have specific formatting all throughout the text.

eg a MESSAGE, and a _Subscriber_.  I would not change that to first occurrence only.

**Comment by oberstet** [comment]
That being said: we should write down the rules, and then stick to it and apply consistently. Whatever we decide ..
